openssl/src/ssl/bio.rs

```diff
@@ -69,6 +71,16 @@ pub unsafe fn get_mut<'a, S: 'a>(bio: *mut BIO) -> &'a mut S {
     &mut state(bio).stream
 }
 
+pub unsafe fn set_dtls_mtu_size<S>(bio: *mut BIO, mtu_size: usize) {
+    const C_LONG_SIZE: usize = std::mem::size_of::<c_long>() * 8;
+    assert!(C_LONG_SIZE <= 64);
+    const C_LONG_MAX: u64 = (1u64 << (C_LONG_SIZE - 1)) - 1;
+    if mtu_size as u64 > C_LONG_MAX {
```

Comment:
This can all just be simplified to `if mtu_size as u64 > c_long::max_value() as u64`

Reply:
Where does `c_long::max_value()` declared? I can't find it in libc's doc.

Reply:
It's on [`i64`](https://doc.rust-lang.org/std/primitive.i64.html#method.max_value), which [`c_long`](https://docs.rs/libc/0.2.65/libc/type.c_long.html) is just an alias for.